### DISCUSSION TITLE
Spawning A Event From The Debug Menu

### DISCUSSION THREAD

**Discussion by PWJDM** [TEHE-Studios/ExpandedHelicopterEvents#334] [Q&A]
Where inside the debug menu do i spawn a Heli from? i've seen things asking about it but no good answers. i dont know where to find the menu to spawn them

**Comment by Chuckleberry-Finn** [comment]
The debug menu is accessible setting `-debug` in your launch options.

**Reply by PWJDM** [reply]
> The debug menu is accessible setting `-debug` in your launch options.

But where inside the debug menu. that was my question i cant find the event spawner

**Reply by Chuckleberry-Finn** [reply]
There is an option called "EHE Debug Tests".
